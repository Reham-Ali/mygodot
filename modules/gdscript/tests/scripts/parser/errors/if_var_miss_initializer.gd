func test():
    if var x:
        pass
