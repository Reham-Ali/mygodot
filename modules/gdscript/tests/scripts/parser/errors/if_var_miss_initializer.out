GDTEST_PARSER_ERROR
Expected type after ":"
